main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"encoding/json"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+	"net/http"
+	"strings"
+	"topokube/internal/namespaces"
+	"topokube/internal/pods"
+)
+
+func apiRoot(w http.ResponseWriter, r *http.Request) {
+	client, err := createClient()
+
+	if err != nil {
+		http.Error(w, "failed to connect to kubernetes", http.StatusInternalServerError)
+	}
+
+	namespaceList := namespaces.ListNamespaces(client)
+
+	json.NewEncoder(w).Encode(namespaceList)
+}
+
+func health(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte("pong"))
+}
+
+func getPods(w http.ResponseWriter, r *http.Request) {
+	pathParts := strings.Split(r.URL.Path, "/")[1:]
+	namespace := pathParts[len(pathParts) - 1]
```

Comment:
I'm intentionally deferring pulling in a router like Chi or mux, so I'm using this primitive approach for now.